src/test/java/org/folio/consortia/support/EntityUtils.java

```diff
@@ -313,7 +313,7 @@ public static TenantCollection createTenantCollection(List<Tenant> tenants) {
 
   public static PublicationRequest createPublicationRequest(ObjectNode payload, HttpMethod method) {
     if (payload.has("source")) {
-      payload.put("source", "consortium");
+      payload.put("source", "CONSORTIUM");
```

Comment:
so we could not use generic method now, need to move logic to set payload into appropriate class.
For settings: source: "consortium"
For policies: source: "CONSORTIUM"
For roles: type: "CONSORTIUM"